tdrs-frontend/src/components/Header/Header.jsx

```diff
@@ -3,6 +3,10 @@ import { useSelector } from 'react-redux'
 import closeIcon from 'uswds/dist/img/close.svg'
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
 import { faSignOutAlt, faUserCircle } from '@fortawesome/free-solid-svg-icons'
+import {
+  accountStatusIsApproved,
+  accountIsInReview,
```

Comment:
Is this multiline due to linting line-length? Or just a stylistic preference?

Reply:
it was for line length - i have my editor configured to auto-format based on the project's prettier settings